testing/test_aux.sh

```diff
@@ -15,6 +15,25 @@ echo Lint checks | tee -a $TEST_RESULTS
 cmd/inbuild skip
 echo cmd/inbuild exit code $? | tee -a $TEST_RESULTS
 
+
+# bacext testing
+echo Running bacext test... | tee -a $TEST_RESULTS
```

Comment:
conceptually this is right, but it should be folded in with the cmd/run below, there's no need to do another one just for this. Notice where it adds the "brute" options?  You can/should just say something like:

startup_faux_1_opts="brute bacnet"

quotes are important.

Also, you can't use local/ here (since it only exists locally to you).  Notice how the other tests are done (snippet below) and follow that model.

Reply:
You missed the part about havin gthis run in the same set as brute -- it shouldn't be a separate test.  You can pass multiple options to the same faux-test statement. See my original comment.